documentation/src/main/asciidoc/stories/assembly_hibernate_cache.adoc

```diff
@@ -0,0 +1,16 @@
+[id='hibernate-cache']
+:context: hibernate-cache
+= Using {brandname} as a second-level cache
+
+{brandname} implements the SPI that Hibernate exposes to allow integration with the second-level cache (2LC).
+This means you can use {brandname} to store data from `Session` or `EntityManager` (JPA) operations.
+
+link:https://docs.jboss.org/hibernate/orm/5.2/userguide/html_single/Hibernate_User_Guide.html#caching[Hibernate documentation]
```

Comment:
Maybe we should link to 5.3, or even 6.0.

Reply:
I lupdated the link to "current"